packages/middleware/src/integrations/helpers/resolveDependency.ts

```diff
@@ -1,14 +1,14 @@
-import consola from "consola";
+import { AlokaiContainer } from "../../types";
 
 /**
  * Resolves dependencies based on the current working directory, not relative to this package.
  */
-export function resolveDependency<T>(name: string): T {
+export function resolveDependency<T>(name: string, alokai: AlokaiContainer): T {
```

Comment:
We need to figure out the better name, container is used only to not use context name but both works as a random entities container. I would suggest to add scoped names.